components/input/__tests__/input.browser-test.jsx

```diff
@@ -501,4 +503,62 @@ describe('SLDSInput', () => {
 			expect(fixedTextLeft.textContent).to.equal('$');
 		});
 	});
+
+	describe('Inline Help', () => {
```

Comment:
Could move these markup tests to jest/storybook examples? 

* Uncomment https://github.com/salesforce/design-system-react/blob/master/components/story-based-tests.js and that should give you snapshot PNGs and DOM/markup output of all the input examples. You will want to add an additional example with the Tooltip `isOpen=true`, so you can capture the markup of the tooltip.

I will try to get the comment header on Mocha test files added in the next week, so it's more obvious for everyone.

Reply:
good call!